relay-server/src/endpoints/attachments.rs

```diff
@@ -13,6 +13,25 @@ use crate::extractors::RequestMeta;
 use crate::service::ServiceState;
 use crate::utils;
 
+#[derive(Debug)]
+struct MultipartWrapper(Multipart);
```

Comment:
I've had an open TODO for a while to ditch axum's Multipart and instead use `multer` directly. Since we now need to write a custom extractor, what if we include that change in this PR?

Reply:
Moved to https://github.com/getsentry/relay/issues/2218